Platform/RaptorlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -29,6 +29,9 @@
 #include <Library/SpiFlashLib.h>
 #include <Register/PmcRegs.h>
 #include <Library/VTdLib.h>
+#if PLATFORM_RPLP
```

Comment:
TCC subregions should be removed. Suggest not adding these codes.

Reply:
Updated with TCC subregions removed.